File: 02Fabryka/Students/2016/wdenejko/FabrykaWytworcza/Car.cs
using System;
using System.Collections.Generic;

namespace FabrykaWytworcza
{
	public abstract class Car
	{
		protected string Producent;
		protected string Model;
		protected double PojemnoscSilnika;
		protected List<string> Wlasciwosci = new List<string>();

		public string GetProducent() {
			return Producent;
		}

		public void Prepare() {
			Console.WriteLine(string.Format("Producent: {0}, Marka: {1}, Pojemnosc: {2}, Wlasciwosci:", Producent, Model, PojemnoscSilnika));
			foreach (var dd in Wlasciwosci)
			{
				Console.WriteLine(dd);
			}
		}

		public override string ToString() {
			var result = string.Empty;
			Wlasciwosci.ForEach(d => result += d + ',');
			return string.Format("Producent: {0}, Marka: {1}, Pojemnosc: {2}, Wlasciwosci: {3}", Producent, Model, PojemnoscSilnika, result);
		}
	}
}

File: 03Builder/Students/2018/DabrowskiFilip/builder/src/main/java/builderOrFactory/cars/Car.java
package builderOrFactory.cars;

public abstract class Car {

    public String name;
    public int productionYear;
    public String model;
    public String gearBoxType;

    @Override
    public String toString() {
        return "Car{" +
                "name='" + name + '\'' +
                ", productionYear=" + productionYear +
                ", model='" + model + '\'' +
                ", gearBoxType='" + gearBoxType + '\'' +
                '}';
    }
}
